lib/text/ttml_text_parser.js

```diff
@@ -368,15 +374,21 @@ shaka.text.TtmlTextParser.parseCue_ = function(
 
   // Get other properties if available.
   let regionElement = shaka.text.TtmlTextParser.getElementFromCollection_(
-      cueElement, 'region', regionElements);
+      cueElement, 'region', regionElements, '');
   if (regionElement && regionElement.getAttribute('xml:id')) {
     let regionId = regionElement.getAttribute('xml:id');
     let regionsWithId = cueRegions.filter(function(region) {
       return region.id == regionId;
     });
     cue.region = regionsWithId[0];
   }
-  shaka.text.TtmlTextParser.addStyle_(cue, cueElement, regionElement, styles);
+  let imageElement = shaka.text.TtmlTextParser.getElementFromCollection_(
+      cueElement, 'smpte:backgroundImage', metadataElements, '#');
+  shaka.text.TtmlTextParser.addStyle_(cue,
```

Comment:
Minor style nit, lately we have been putting all the arguments on the same line:

```
shaka.text.TtmlTextParser.addStyle_(
    cue, cueElement, ... , styles);
```

or start the vertical list on the next line:

```
shaka.text.TtmlTextParser.addStyle_(
    cue,
    cueElement,
    ... ,
    styles);
```

This makes it easier to update if/when the method name changes because the argument alignment won't change.